packages/portal/src/components/set/AddItemToSetModal.vue

```diff
@@ -99,11 +99,22 @@
     },
 
     methods: {
-      fetchCollections() {
-        this.$store.dispatch('set/fetchCreations')
-          .then(() => {
-            this.fetched = true;
-          });
+      async fetchCollections() {
+        const creator = this.$auth.user?.sub;
+        // TODO: pagination and/or search within one's collections
+        const searchParams = {
+          query: `creator:${creator}`,
+          profile: 'standard',
+          pageSize: 100,
+          page: 0,
+          qf: [
+            'type:Collection'
+          ]
+        };
+
+        const searchResponse = await this.$apis.set.search(searchParams, { withMinimalItemPreviews: true });
+        this.collections = searchResponse.data.items;
```

Comment:
I'm thinking we want to add an empty array fallback here for when there are no sets returned. At the moment I don't see it might break the code if `collections` will become undefined, but it will be more as expected as the default data prop is an empty array too.

Reply:
OK. Reverted.